### PR TITLE
aria-labelledby example for image has accessible name (23a2a8)

### PR DESCRIPTION
**Pull request by Jym77**
* Add an example where accessible name is given by `aria-labelledby`.
* Add final period at the end of example descriptions.
* Unify a bit example descriptions.

(Ouch github diff tool is mixing things up… The new example is Passed Example 3, the rest has been pushed down but github still tries to compare old Passed Example 4 with new Passed Example 4 instead of new Passed Example 5…)

This will require a new Final Call as new example is added.

Closes issue(s): 
- #830 

----

### **When creating PR:**
- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**
- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/image-has-name-23a2a8.md [hunk 1]
```diff
@@ -53,55 +53,64 @@ There is a known combination of a popular browser and assistive technology that
...
 ```html
-<img title="W3C logo" />
+<div style="display: hidden" id="img-label">W3C logo</div>
+<div role="img" aria-labelledby="img-label"></div>
 ```
 
 #### Passed Example 4
 
-The HTML `img` element is marked as [decorative](#decorative) through an empty `alt` attribute
+The HTML `img` element has an accessible name, though the name is not always accessibility supported.
```
Comment (audreymaniez) [inline _rules/image-has-name-23a2a8.md]: Just a suggestion: to be consistent with your other descriptions add the "given by the `title` attribute." information ?

And maybe replace "the name" by "the `title`" ? 


> The HTML `img` element has an accessible name given by a `title` attribute, though the `title` is not always accessibility supported.

File: _rules/image-has-name-23a2a8.md [hunk 2]
```diff
@@ -53,55 +53,64 @@ There is a known combination of a popular browser and assistive technology that
...
+The element with a [semantic role](#semantic-role) of `img` has an accessible name given by the `aria-label` attribute.
 
 ```html
 <div role="img" aria-label="W3C logo"></div>
 ```
 
 #### Passed Example 3
 
-The element has an accessible name, though the name is not always accessibility supported
+The element with a [semantic role](#semantic-role) of `img` has an accessible name given by an `aria-labelledby` attribute and element.
```
Comment (audreymaniez) [inline _rules/image-has-name-23a2a8.md]: I think "and element" at the end of the description is useless ? 

> The element with a [semantic role](#semantic-role) of `img` has an accessible name given by an `aria-labelledby` attribute.

File: _rules/image-has-name-23a2a8.md [hunk 3]
```diff
@@ -53,55 +53,64 @@ There is a known combination of a popular browser and assistive technology that
...
 
 ```html
-<img title="W3C logo" />
+<div style="display: hidden" id="img-label">W3C logo</div>
```
Comment (audreymaniez) [inline _rules/image-has-name-23a2a8.md]: It seems `display:hidden` doesn't exist.

I can't find in this documentation : https://drafts.csswg.org/css-display/#the-display-properties

But I may be wrong, I'm not a CSS expert 😇 

I suggest using either use `display:none` or `visibility:hidden` 

```html

<div style="display: none" id="img-label">W3C logo</div>

**Comment by jkodu** [comment]
Hi @Jym77, 

We (myself & @WilcoFiers) reverted this merge from develop branch. 

But this leaves you with more work. You will have to branch from this pull request and create a new PR (as this cannot be reopened). 

And re-request approvals. I will let you do that side of this. Thanks.

**Comment by jkodu** [comment]
@Jym77 - I also closed https://github.com/act-rules/act-rules.github.io/pull/849 as it is not necessary anymore.

**Pull request merged**